homeassistant/util/unit_system.py

```diff
@@ -43,6 +46,8 @@
 
 LENGTH_UNITS = DistanceConverter.VALID_UNITS
 
+AREA_UNITS = AreaConverter.VALID_UNITS
+
```

Comment:
I don't think this is needed
```suggestion
AREA_UNITS = AreaConverter.VALID_UNITS

```